packages/cw721-init-msg-registry/src/contracts/sg721.rs

```diff
@@ -0,0 +1,21 @@
+use cosmwasm_std::{to_binary, Binary, StdResult};
+
+use crate::data::InitMsgData;
+
+/// Generate sg721 init message given ics721 data
+pub fn ics721_get_init_msg(data: &InitMsgData) -> StdResult<Binary> {
```

Comment:
We're also going to need at least one unit test that this works the way that it is intended.